racetime/views/user.py

```diff
@@ -134,6 +134,37 @@ def get(self, request, *args, **kwargs):
         return resp
 
 
+class UserProfileData(ViewProfile):
+    def get(self, request, *args, **kwargs):
+        self.object = self.get_object()
+        user = self.object.api_dict_summary()
+        entrances = self.get_entrances()
+        paginator = Paginator(entrances, 10)
```

Comment:
Unused variable. 🙂

Reply:
This is what i get for not running all my linters against the codebase, didn't want to cause some havoc though.

Reply:
I'm mostly cool with code linting in PRs, assuming it's not too crazy lol